django/contrib/admin/helpers.py

```diff
@@ -83,8 +83,8 @@ def media(self):
             extra = '' if settings.DEBUG else '.min'
             js = [
                 'vendor/jquery/jquery%s.js' % extra,
-                'jquery.init.js',
                 'collapse%s.js' % extra,
+                'jquery.init.js',
```

Comment:
@apollo13 I just wanted to see, if I it will keep the order. It does, so I can run all scripts in between with jQuery and disable it at the end. Even if there are multiple widgets using the buildin jQuery, they will have access to it, before it is disabled at the end.

So we could keep `jQuery.noConflict(true)` but do not have to create our own version Select2. Does would that work for you?